src/actions/channels.test.js

```diff
@@ -1550,7 +1550,7 @@ describe('Actions.Channels', () => {
 
     it('updateChannelHeader', async () => {
         nock(Client4.getChannelsRoute()).
-            post(`${TestHelper.basicChannel.id}`).
+            get(`/${TestHelper.basicChannel.id}`).
```

Comment:
Why this change?

Reply:
The getChannel API wasn't mocked properly before since it uses a GET request, not a POST. The test still passed because of the bug that this PR fixes, so we had to fix the mock to be able to fix the bug.